src/MergeContext.js

```diff
@@ -243,7 +243,17 @@ class MergeContext {
         const tagCommit = await GH.getCommit(this._tagSha);
         const prMergeSha = await GH.getReference(this._mergePath());
         const prCommit = await GH.getCommit(prMergeSha);
-        return tagCommit.tree.sha === prCommit.tree.sha;
+        const result = tagCommit.tree.sha === prCommit.tree.sha;
+        this._log("tag freshness: " + result);
+        return result;
+    }
+
+    // Whether the PR message has not changed since the PR staged commit creation.
```

Comment:
Please use `Whether the commit message configuration remained intact since staging.`